src/main/java/com/microsoft/cse/helium/app/config/WebConfig.java

```diff
@@ -0,0 +1,11 @@
+package com.microsoft.cse.helium.app.config;
```

Comment:
This file is completely commented out. Does it need to be there?

Reply:
It is work in progress.  It may be removed in final form for handling robots.  However, this PR does not address robots.